examples/maps/composite_map_AIA_HMI.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""
+=============
+Composite map
+=============
+In this example we make a composite map out of images provided by AIA and HMI.
```

Comment:
Can you expand this description a little bit please. It should explain what the example is doing, i.e overplotting the magnetic field on a EUV image etc...

Reply:
I extended the description. Check later if that was what you had in mind.